packages/rich-text-editor/src/RichTextEditor/RichTextEditor.module.scss

```diff
@@ -52,6 +52,13 @@
   background: $color-white;
   border-radius: $border-solid-border-radius;
   word-wrap: break-word;
+
+  &:hover {
```

Comment:
Do we want to also add a focus-within on that one too? Depends whether we want to remove the darker for keyboard users or not? @jmitsios might need a quick consult on this one 🙏 
![Screen Shot 2022-08-01 at 5 52 00 pm](https://user-images.githubusercontent.com/36558508/182100161-6877415e-8e8c-451a-830d-39612a4175e6.png)
![Screen Shot 2022-08-01 at 5 51 24 pm](https://user-images.githubusercontent.com/36558508/182100177-6b6475a6-5c22-4e93-9568-5f817e19522c.png)

Reply:
@mcwinter07 — Just to make sure I'm following along here, is the question about whether we want all the borders to still darken to Gray-600 for keyboard users, when placing focus on the text input area?  (I'm not too sure what focus-within is, sorry 😞)

Reply:
@jmitsios no worries, to clarify - if the user is focused on the input (1st screenshot) should the entire border be Gray-600 like on hover (2nd screenshot). Currently it has the focus ring but not the darkened border

Reply:
Ahh! I see what you mean now.  I believe that most of our components have their hover states triggered upon focusing, as it's seen as the 'keyboard equivalent' of hovering over it.   So in this instance, it feels like it makes sense to trigger the borders when the focus ring is active.

Reply:
Cool beans, thanks for confirming! @JakePitman so we should be able to chuck a `&:focus-within` to action that :)